src/bindings/python/src/pyopenvino/graph/passes/pattern_ops.cpp

```diff
@@ -553,25 +550,61 @@ static void reg_pattern_optional(py::module m) {
         }),
         py::arg("type_names"),
-        py::arg("input"),
+        py::arg("inputs"),
         R"(
         Create Optional with the given node type and input node.
 
         :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
         :type type_names: List[str]
 
+        :param inputs: input node's output vector.
+        :type inputs: List[openvino.runtime.Output]
+    )");
+
+    optional_type.def(py::init([](const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input) {
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::as_output_vector({input}),
+                                                                                   nullptr);
+                      }),
+                      py::arg("type_names"),
+                      py::arg("input"),
+                      R"(
+        Create Optional with the given node type and input node.
+
+        :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
+        :type type_names: List[str]
+
         :param input: input node
         :type input: openvino.runtime.Node
     )");
 
+    optional_type.def(py::init([](const std::vector<std::string>& type_names, const ov::NodeVector& inputs) {
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::as_output_vector(inputs),
+                                                                                   nullptr);
+                      }),
+                      py::arg("type_names"),
+                      py::arg("input"),
+                      R"(
+        Create Optional with the given node type and input node.
+
+        :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
+        :type type_names: List[str]
+
+        :param inputs: input node vector
+        :type inputs: List[openvino.runtime.Node]
+    )");
+
     optional_type.def(py::init([](const std::vector<std::string>& type_names,
                                   const std::shared_ptr<ov::Node>& input,
                                   const Predicate& pred) {
-                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names), input, pred);
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::as_output_vector({input}),
+                                                                                   pred);
                       }),
                       py::arg("type_names"),
                       py::arg("input"),
```

Comment:
Could you please also update `py::arg("pred"),` -> `py::arg("predicate"),`.
I believe, we missed it from prev PR

Reply:
Fixed, thanks!